www/include/common/mobile_menu.php

```diff
@@ -12,11 +12,30 @@
             <?php if (!empty($subMenu['children'])) : ?>
                 <ul class="nav-items nav-expand-content">
                 <?php foreach ($subMenu['children'] as $index2 => $subMenu2) : ?>
-                    <li class="nav-item">
-                        <a class="nav-link" href="main.php?p=<?= substr($index2, 1) ?>">
-                            <?= $subMenu2['label'] ?>
-                        </a>
-                    </li>
+                    <?php if (!empty($subMenu2['children'])) : ?>
+                        <li class="nav-item nav-expand">
```

Comment:
```suggestion
                        <li class="nav-item nav-expand-content">
```

Reply:
please check the class consistency ;)

Reply:
If I do this, the third level is broken ;)